config/settings/local.py

```diff
@@ -11,3 +13,12 @@
 ASYNC_FILE_UPLOAD = False
 
 AXES_ENABLED = False
+
+INSTALLED_APPS.append("debug_toolbar")
+MIDDLEWARE.insert(0, "debug_toolbar.middleware.DebugToolbarMiddleware")
+INTERNAL_IPS = [
+    "127.0.0.1",
+]
+DEBUG_TOOLBAR_CONFIG = {
+    "SHOW_TOOLBAR_CALLBACK": lambda request: True,
```

Comment:
I know the default check doesn't work as we don't supply the docker IP to `INTERNAL_IPS` but we can still check that we are running in DEBUG mode.

```suggestion
    "SHOW_TOOLBAR_CALLBACK": lambda request: DEBUG,
```